### PR TITLE
add readme improvement from pirey pull request

### PR DESCRIPTION
**Pull request by shintasapta**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+* [Redis](https://redis.io/)
+
+## Installation
+
+clone project :
+
+* Dengan https :
+
+```
+git clone https://gitlab.com/ajaro-id/simdes/simdes-api.git
```
Comment (andhikayuana) [inline README.md]: @shintasapta iki koq sih gitlab ? padahal wis tak review loh 🤔

File: README.md [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+```
+
+* Dengan ssh :
+
+```
+git clone git@gitlab.com:ajaro-id/simdes/simdes-api.git
```
Comment (andhikayuana) [inline README.md]: @shintasapta iki jg

Comment (shintasapta) [reply README.md]: oke segera perbaiki

File: README.md (hunk 1 above)
Comment (shintasapta) [reply README.md]: btw iki ning working directory ku wes tak ganti github

**Comment by andhikayuana** [comment]
nice, makasih @shintasapta

**Pull request merged**